internal/kafka/internal/workers/cluster_mgrs/dynamic_scaleup_mgr.go

```diff
@@ -57,14 +61,18 @@ func (m *DynamicScaleUpManager) Stop() {
 }
 
 func (m *DynamicScaleUpManager) Reconcile() []error {
```

Comment:
If we want we can add a return at the beginning of this method so we can merge safely and iterate on that

Reply:
I modified the condition at the beginning to always be true in order to do that